src/olympia/hero/serializers.py

```diff
@@ -40,10 +39,7 @@ def get_description(self, obj):
         elif obj.description:
             return ugettext(obj.description)
         else:
-            addon = obj.promoted_addon.addon
-            return (str(addon.summary)
-                    if addon.summary and addon.type == amo.ADDON_EXTENSION
-                    else '')
+            return str(obj.promoted_addon.addon.summary or '')
```

Comment:
Presumably this entire chain of props always exists?

Reply:
yes - both PrimaryHero.promoted_addon, and PromotedAddon.addon are defined with `null=False` so at the database level they must exist.